test/elements.test.ts

```diff
@@ -0,0 +1,166 @@
+import { ElementDictionary } from '../src/data/elements';
+import { Phases, Phase } from '../src/interfaces/element-props';
+
+const elements = [
+  'Hydrogen',
+  'Helium',
+  'Lithium',
```

Comment:
No need to explicitly define this array. If you need it for testing, you can simply do `Object.keys(ElementDictionary)`.